addons/sale_project/models/product_template.py

```diff
@@ -47,73 +47,63 @@ def _compute_service_policy(self):
+                    record.product_tooltip += _(
                         "Create a project for the order with a task for each sales order line "
                         "to track the time spent."
                     )
                 elif record.service_tracking == 'task_in_project':
-                    record.product_tooltip = _(
-                        "Invoice ordered quantities as soon as this service is sold. "
+                    record.product_tooltip += _(
                         "Create an empty project for the order to track the time spent."
                     )
             elif record.service_policy == 'delivered_milestones':
-                if record.service_tracking == 'no':
-                    record.product_tooltip = _(
-                        "Invoice your milestones when they are reached."
-                    )
-                elif record.service_tracking == 'task_global_project':
-                    record.product_tooltip = _(
-                        "Invoice your milestones when they are reached. "
+                record.product_tooltip = _(
+                    "Invoice your milestones when they are reached. "
+                )
+                if record.service_tracking == 'task_global_project':
+                    record.product_tooltip += _(
                         "Create a task in an existing project to track the time spent."
                     )
                 elif record.service_tracking == 'project_only':
-                    record.product_tooltip = _(
-                        "Invoice your milestones when they are reached. "
+                    record.product_tooltip += _(
                         "Create a project for the order with a task for each sales order line "
                         "to track the time spent."
                     )
                 elif record.service_tracking == 'task_in_project':
-                    record.product_tooltip = _(
-                        "Invoice your milestones when they are reached. "
+                    record.product_tooltip += _(
                         "Create an empty project for the order to track the time spent."
                     )
             elif record.service_policy == 'delivered_manual':
-                if record.service_tracking == 'no':
-                    record.product_tooltip = _(
-                        "Invoice this service when it is delivered (set the quantity by hand on your sales order lines). "
-                    )
-                elif record.service_tracking == 'task_global_project':
-                    record.product_tooltip = _(
-                        "Invoice this service when it is delivered (set the quantity by hand on your sales order lines). "
+                record.product_tooltip = _(
+                    "Invoice this service when it is delivered (set the quantity by hand on your sales order lines). "
+                )
```

Comment:
Same remark, don't try to extract a shared part, it's easier for translators to have a full sentence, the sentence order and construction differ between languages.